addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3608,7 +3608,7 @@ msgid "Singles"
 msgstr ""
 
 msgctxt "#1051"
-msgid "Enter web address"
+msgid "Enter Web address"
```

Comment:
It is - World Wide Web in any reputable dictionary. 
http://en.wikipedia.org/wiki/World_Wide_Web
http://www.merriam-webster.com/dictionary/world%20wide%20web

So anything with Web interface, WebServer Webserver Web page etc etc is correct.